ledger/common/bitutils/utils_test.go

```diff
@@ -46,16 +58,20 @@ func TestBitTools(t *testing.T) {
 
 	t.Run("SetBit", func(t *testing.T) {
 		b.SetInt64(0)
-		bytes := make([]byte, (maxBits+7)/8) // length in bytes needed for maxbits
+		bytes := MakeBitVector(maxBits)
+		rand.Read(bytes) // fill bytes with random garbage to verify that setting each individual bit works
+
 		// build a random big bit by bit
-		for i := 0; i < maxBits; i++ {
+		for idx := 0; idx < maxBits; idx++ {
 			bit := rand.Intn(2)
 			// b = 2*b + bit
 			b.Lsh(&b, 1)
 			b.Add(&b, big.NewInt(int64(bit)))
 			// sets the bit at index i
 			if bit == 1 {
-				SetBit(bytes, i)
+				SetBit(bytes, idx, 1)
```

Comment:
any reasion this isn't just `SetBit(bytes, idx, bit)`?

Reply:
Good observation! This is unnecessarily complex: 
https://github.com/onflow/flow-go/blob/aa285f72ff6cbb8c601178e909eb9f86bccfe9a9/ledger/common/bitutils/utils_test.go#L71-L75
Replaced it by the following code, which is much more compact 
https://github.com/onflow/flow-go/blob/3426d41a814a6f658e23363e6281e0702fa02a9a/ledger/common/bitutils/utils_test.go#L70